astropy/constants/CODATA2010.py

```diff
@@ -0,0 +1,125 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Astronomical and physics constants in SI units.  See :mod:`astropy.constants`
+for a complete listing of constants defined in Astropy.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+
+from .constant import Constant
+
+
+# PHYSICAL CONSTANTS
+
+class Codata2010(Constant):
+    default_reference = 'CODATA 2010'
+    _registry = {}
+    _has_incompatible_units = set()
+
+    def __new__(cls, abbrev, name, value, unit, uncertainty,
+                reference=default_reference, system=None):
+        return(super(Codata2010, cls).__new__(cls, abbrev, name, value, unit, uncertainty,
+                        reference, system))
+
+
+class EMCodata2010(Codata2010):
+    """An electromagnetic constant."""
+
+    @property
```

Comment:
But I realized something: this `EMCodata2010` will have its own registry, which is not, I think, what we want. So, instead of `pass`, write
```
    _registry = Codata2010._registry
```